app/shared/favorite-resources/FavoriteResourcesContainer.js

```diff
@@ -0,0 +1,42 @@
+import React, { PropTypes } from 'react';
+import Loader from 'react-loader';
+import { connect } from 'react-redux';
+
+import { fetchResources, unfavoriteResource } from 'api/actions';
+import selector from './favoriteResourcesSelector';
+import FavoriteResources from './FavoriteResources';
+
+export class UnconnectedFavoriteResourcesContainer extends React.Component {
+  static propTypes = {
+    fetchResources: PropTypes.func.isRequired,
+    isFetching: PropTypes.bool.isRequired,
+    resources: PropTypes.array.isRequired,
+    unfavoriteResource: PropTypes.func.isRequired,
+  };
+
+  componentDidMount() {
+    // Make sure resources have correct isFavorite value by fetching
+    // them. When resources are fetched on app initialization, isFavorite can
+    // be false even though resource is users's favorite (probably because
+    // resources are fetched before current user is known).
```

Comment:
- [ ] It'd be nicer to delay fetching the resources etc. in the app container until the user is known. That'd also avoid other similar problems in the future.